internal/grpc/common.go

```diff
@@ -0,0 +1,43 @@
+package grpc
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+
+	"github.com/mikewurtz/taskman/certs"
+)
+
+// LoadCACertPool loads the CA certificate pool from the embedded files
+func LoadCACertPool() (*x509.CertPool, error) {
+	caCert, err := certs.CertFiles.ReadFile(certs.CACertFileName)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read CA cert: %w", err)
+	}
+
+	pool := x509.NewCertPool()
+	if !pool.AppendCertsFromPEM(caCert) {
+		return nil, fmt.Errorf("failed to append CA certificate")
```

Comment:
```suggestion
		return nil, errors.New("failed to append CA certificate")
```

When you don't have any format verbs and just need an error with a static message you should prefer `errors.New`.

Reply:
Good idea, I will update the rest of the spots like this